src/languages/r.js

```diff
@@ -66,12 +112,10 @@ export default function(hljs) {
       },
 
       {
-        className: 'string',
-        contains: [hljs.BACKSLASH_ESCAPE],
-        variants: [
-          {begin: '"', end: '"'},
-          {begin: "'", end: "'"}
-        ]
+        // infix operator
+        begin: '%',
+        end: '%',
+        relevance: 10
```

Comment:
Also a lot of things have % so probably doesn't deserve any relevance boost.

Reply:
Yeah, that actually led to detection failures and is already removed — I just forgot to push those changes.